jena-arq/Grammar/RDF-Protobuf/README

```diff
@@ -1,6 +1,20 @@
 ## Licensed under the terms of http://www.apache.org/licenses/LICENSE-2.0
 
+An Apache Jena reklease includes the java generated by teh protobuf compiler.
```

Comment:
s/reklease/release

s/teh/the

Reply:
Ug - classic! - failed to write the editor buffer.